<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CS2饰品交易管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_inventory">
       <attribute name="title">
        <string>库存管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_add">
            <property name="text">
             <string>添加商品</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="filter_group">
          <property name="title">
           <string>筛选器</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="name_filter_layout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>商品名称:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="name_filter">
               <property name="minimumWidth">
                <number>200</number>
               </property>
               <property name="placeholderText">
                <string>输入商品名称</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="type_filter_layout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>商品类型:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="type_filter">
               <property name="minimumWidth">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="subtype_filter">
               <property name="minimumWidth">
                <number>120</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="wear_filter_layout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>磨损等级:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="wear_filter">
               <property name="minimumWidth">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="state_filter_layout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>状态:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="state_combo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="price_filter_layout">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>价格范围:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="price_min">
               <property name="minimumWidth">
                <number>100</number>
               </property>
               <property name="prefix">
                <string>¥</string>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="price_max">
               <property name="minimumWidth">
                <number>100</number>
               </property>
               <property name="prefix">
                <string>¥</string>
               </property>
               <property name="maximum">
                <double>999999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_clear_filter">
             <property name="text">
              <string>清空筛选</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="inventory_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_sold">
       <attribute name="title">
        <string>已售商品</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="sold_items_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_analytics">
       <attribute name="title">
        <string>统计分析</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="stats_group">
          <property name="title">
           <string>总体统计</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="stats_layout">
             <item>
              <widget class="QLabel" name="total_investment_label">
               <property name="text">
                <string>总投资: ¥0.00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="total_value_label">
               <property name="text">
                <string>总市值: ¥0.00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="total_profit_label">
               <property name="text">
                <string>总收益: ¥0.00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="total_sales_label">
               <property name="text">
                <string>总销售额: ¥0.00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="remaining_balance_label">
               <property name="text">
                <string>剩余余额: ¥0.00</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="item_count_label">
               <property name="text">
                <string>持有数量: 0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="sold_count_label">
               <property name="text">
                <string>已售数量: 0</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="avg_profit_label">
               <property name="text">
                <string>平均收益: ¥0.00</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="buttons_layout">
             <item>
              <widget class="QPushButton" name="btn_adjust_investment">
               <property name="text">
                <string>调整投资额</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="period_layout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>时间段:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="period_combo"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="analytics_table"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
